packages/itwinui-react/src/core/Tooltip/Tooltip.tsx

```diff
@@ -16,46 +54,141 @@ export type TooltipProps = {
    * Element to have tooltip on. Has to be a valid JSX element and needs to forward its ref.
    * If not specified, the `reference` prop should be used instead.
    */
-  children?: JSX.Element;
-} & Omit<PopoverProps, 'className'> &
-  Omit<CommonProps, 'title'>;
+  children?: React.ReactNode;
+  /**
+   * Element to portal tooltip to.
+   * Portals to ThemeProvider portalContainerRef by default.
+   */
+  portal?: { to: HTMLElement };
+};
 
-/**
- * Basic tooltip component to display informative content when an element is hovered or focused.
- * Uses the {@link Popover} component, which is a wrapper around [tippy.js](https://atomiks.github.io/tippyjs).
- * @example
- * <Tooltip content='tooltip text' placement='top'><div>Hover here</div></Tooltip>
- * @example
- * const buttonRef = React.useRef();
- * ...
- * <Button ref={buttonRef} />
- * <Tooltip content='tooltip text' reference={buttonRef} />
- */
-export const Tooltip = (props: TooltipProps) => {
-  const { content, children, className, style, visible, ref, id, ...rest } =
-    props;
+const useTooltip = ({
+  placement = 'top',
+  visible: controlledOpen,
+  toggleVisible: setControlledOpen,
+  followTrigger = false,
+}: TooltipOptions = {}) => {
+  const [uncontrolledOpen, setUncontrolledOpen] = React.useState(false);
+
+  const open = controlledOpen ?? uncontrolledOpen;
+  const setOpen = setControlledOpen ?? setUncontrolledOpen;
+
+  const data = useFloating({
+    placement,
+    open,
+    onOpenChange: setOpen,
+    whileElementsMounted: (referenceEl, floatingEl, update) =>
+      autoUpdate(referenceEl, floatingEl, update, {
+        animationFrame: followTrigger,
+      }),
+    middleware: [offset(5), flip(), shift()],
```

Comment:
i think we were using `4` as offset before

Reply:
Updated, will add user overrides in future commit